### DISCUSSION TITLE
BUG:Tabs title switching bug

### DISCUSSION THREAD

**Discussion by 110011100101** [GitbookIO/community#355] [Feature Requests]
When there is more than one tabs in doc, the tab's title in preview will always switch together.
If you want to repetition it,there is something you need to know:
-- It only happen in preview ( the state that cannot be compose (after click merge?) )
-- It only happen at the first time you open it (preview)
-- I haven't test if it will happen with more than two tabs in a doc yet.
-- I haven't test this bug in other doc.

**Comment by SamyPesse** [comment]
It's not a bug, but it was done by design. It was designed to provide a better experience when using tabs to document something like an SDK with multi-language support, where the reader selects an option in the first tab, and the other tabs are kept in sync:

<img width="793" alt="Screenshot 2023-10-03 at 15 46 54" src="https://github.com/GitbookIO/community/assets/845425/7d1d8942-181a-4428-9e51-170277861bd4">

I'm curious how you are using tabs and how you think this behavior can conflict with what you are trying to achieve.

**Reply by 110011100101** [reply]
Oh~,I know what u say.
I use tabs to store some game content of a module so that I don't have to scroll up and down the doc.
Like a skill,they all have level title.
<img width="416" alt="image" src="https://github.com/GitbookIO/community/assets/113879025/1f640007-66cb-4d8b-b006-905e089bec3f">
<img width="461" alt="image" src="https://github.com/GitbookIO/community/assets/113879025/29e7d3e3-bec7-4729-9829-d78a06a65887">
This is not a big problem,does not affect the use.
